src/convert.cpp

```diff
@@ -1569,18 +1569,19 @@ constexpr ConvFctList convFctList[] = {
     // Update the convertStringCharset() documentation if you add more here!
 };
 
-[[maybe_unused]] bool convertStringCharsetWindows(std::string& str, const char* from, const char* to) {
+bool convertStringCharsetWindows(std::string& str, std::string_view from, std::string_view to) {
```

Comment:
```suggestion
[[maybe_unused]] bool convertStringCharsetWindows(std::string& str, std::string_view from, std::string_view to) {
```
This change seems unrelated.

Reply:
Yes and no. This is already ifdef’d properly.